spec/lib/section_reslugger_spec.rb

```diff
@@ -31,7 +31,7 @@
       body: 'section-edition-body'
     )
     manual_record.editions.create!(
-      section_ids: [
+      section_uuids: [
         'section-id'
```

Comment:
I consciously didn't rename them in this commit. There are quite a few variables/methods in the code that still refer to `id` instead of `uuid` (when dealing with `Section`s) and I plan to open a separate PR to try to rename all those up together.